http-server/src/main/java/org/triplea/server/user/account/PasswordBCrypter.java

```diff
@@ -18,7 +19,7 @@
   public String apply(final String password) {
     // TODO: Md5-Deprecation remove hashing here, move to client-side
     final String hashed = hashPasswordWithSalt(password);
-    return BCrypt.hashpw(hashed, BCrypt.gensalt());
+    return BCrypt.with(LongPasswordStrategies.none()).hashToString(10, hashed.toCharArray());
```

Comment:
Really similar to my other comment, I think we may benefit to create a utility method around this hasing method:
- provides a good place to document decisions/parameters:
  - why '10'?
  - why 'LongPasswordStrategies.none()'?
- Avoids repetition between source code and test (DRY)